### ISSUE TITLE
[Feature] `waydroid prop set persist.waydroid.multi_windows` should be `true` by default.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [waydroid/waydroid#1517]
### Is your feature request related to a problem? Please describe.

Most users want to run specific applications, and interact with them as they would any other. That was mostly why the Windows Subsystem for Android was as popular as it was, despite its lack of the Google Play Store.

However, as [this Reddit post](https://www.reddit.com/r/waydroid/comments/124olm1/comment/je1g0rj/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Waydroid%20docs%20have%20information%20on%20how%20to%20enable%20this:%20https://docs.waydro.id/usage/install%2Don%2Ddesktops%23launch%2Dwaydroid%2Din%2Dmulti%2Dwindow%2Dmode) explains, `waydroid.multi_windows` isn't discoverable.

Nor should it need to be. WayDroid creates independent `.desktop` files in the DE's Start Menu equivalent for both the main AOSP GUI and each application. For the application's icons to not launch solely themselves is actually the less intuitive option.

### Describe the solution you'd like.

Set `waydroid prop set persist.waydroid.multi_windows` by default.

### Describe alternatives you've considered.

I've not considered any of worth (at least, which wouldn't require something equivalent to a game launcher, of sorts).

**Comment by abtye** [comment]
I disagree with you. The desktop environment of Linux is much more complex than that of Windows. If any problems arise, as you said, waydroid.multi_windows is not discoverable easily, users may not know how to close it. This will bring unnecessary trouble.
﻿
﻿

**Comment by RokeJulianLockhart** [comment]
> The desktop environment of Linux is much more complex than that of Windows.

@abtye, what do you refer to? The fact that two compositors exist?

**Comment by abtye** [comment]
There is a third-party project [`Waydroid-Helper`](https://github.com/ayasa520/waydroid-helper) that can configure Waydroid in a GUI way. Maybe the Waydroid official can add it to the [Community Projects We Like](https://docs.waydro.id/faq/community-projects-we-like) in the documentation.
